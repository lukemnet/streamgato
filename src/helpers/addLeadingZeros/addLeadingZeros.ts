const addLeadingZeros = (value: number) =>
  String(value).padStart(2, '0');

export default addLeadingZeros;
