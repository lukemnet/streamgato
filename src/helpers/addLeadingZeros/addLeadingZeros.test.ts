import addLeadingZeros from './addLeadingZeros';

const testValues = [ 0, 1, 2, 3, 40, 45, 60 ];

it('should add leading zeros to numbers', () => {
  testValues.forEach(value => {
    expect(addLeadingZeros(value)).toMatchSnapshot();
  });
});

it('should return string', () => {
  testValues.forEach(value => {
    expect(typeof addLeadingZeros(value)).toBe('string');
  });
});
