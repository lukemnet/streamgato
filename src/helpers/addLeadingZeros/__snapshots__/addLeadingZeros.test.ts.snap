// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should add leading zeros to numbers 1`] = `"00"`;

exports[`should add leading zeros to numbers 2`] = `"01"`;

exports[`should add leading zeros to numbers 3`] = `"02"`;

exports[`should add leading zeros to numbers 4`] = `"03"`;

exports[`should add leading zeros to numbers 5`] = `"40"`;

exports[`should add leading zeros to numbers 6`] = `"45"`;

exports[`should add leading zeros to numbers 7`] = `"60"`;
